### PR TITLE
🍀Add "Softmax Regression"

### LINKED ISSUE #697: Add "softmax-regression"

### PR DESCRIPTION
**Pull request by duythanhvn**
Close #697

### CODE REVIEW

File: chapter_linear-networks/softmax-regression_vn.md [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+<!--
+* Does this email belong in the spam folder or the inbox*?
+* Is this customer more likely *to sign up* or *not to sign up* for a subscription service?*
+* Does this image depict a donkey, a dog, a cat, or a rooster?
+* Which movie is Aston most likely to watch next?
+-->
+
+*dịch đoạn phía trên*
+
+<!-- ===================== Kết thúc dịch Phần 1 ===================== -->
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: @duythanhvn em có thể để các phần dịch lần đầu dài hơn một chút, ngắn quá không bõ công dịch và review. Ví dụ phần 1 này có thể kéo dài đến hết chỗ kết thúc revise phần 1.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+The distinction tends to get blurred, in part, because often, even when we only care about hard assignments, we still use models that make soft assignments.
+-->
+
+*dịch đoạn phía trên*
+
+<!-- ========================================= REVISE PHẦN 1 - KẾT THÚC ===================================-->
+
+<!-- ========================================= REVISE PHẦN 2 - BẮT ĐẦU ===================================-->
+
+<!--
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: Phần 1 đến đây.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+In our case, $y$ would be a three-dimensional vector, with $(1, 0, 0)$ corresponding to "cat", $(0, 1, 0)$ to "chicken" and $(0, 0, 1)$ to "dog".
+-->
+
+*dịch đoạn phía trên*
+
+<!-- ========================================= REVISE PHẦN 2 - KẾT THÚC ===================================-->
+
+<!-- ========================================= REVISE PHẦN 3 - BẮT ĐẦU ===================================-->
+
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: Phần 2 đến đây.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 4]
```diff
@@ -0,0 +1,688 @@
...
+Note that we have gathered all of our weights into a $3\times4$ matrix and that for a given example $\mathbf{x}$, 
+our outputs are given by a matrix-vector product of our weights by our inputs plus our biases $\mathbf{b}$.
+-->
+
+*dịch đoạn phía trên*
+
+<!-- ========================================= REVISE PHẦN 3 - KẾT THÚC ===================================-->
+
+<!-- ========================================= REVISE PHẦN 4 - BẮT ĐẦU ===================================-->
+
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: Phần 3 đến đây.

File: chapter_linear-networks/softmax-regression_vn.md [hunk 5]
```diff
@@ -0,0 +1,688 @@
...
+$$
+
+<!--
+The logits $\mathbf{o}$ then are simply the pre-softmax values that determining the probabilities assigned to each category.
+Summarizing it all in vector notation we get ${\mathbf{o}}^{(i)} = \mathbf{W} {\mathbf{x}}^{(i)} + {\mathbf{b}}$, where ${\hat{\mathbf{y}}}^{(i)} = \mathrm{softmax}({\mathbf{o}}^{(i)})$.
+-->
+
+*dịch đoạn phía trên*
+
+<!-- ========================================= REVISE PHẦN 4 - KẾT THÚC ===================================-->
```
Comment (tiepvupsu) [inline chapter_linear-networks/softmax-regression_vn.md]: Phần 4 đến đây.

Như phần Revise em chia như thế là đẹp.

Nhìn chung, từ giờ em chia lại mỗi phần dịch khoảng bằng với các phần revise hiện tại. Và mỗi phần revise = 2 phần dịch là đẹp.
